app/models/order_item.rb

```diff
@@ -20,20 +20,22 @@ class OrderItem < ApplicationRecord
   validates_presence_of :service_plan_ref
   validates_presence_of :portfolio_item_id
 
+  attribute :workflow_ref, :string
+
   belongs_to :order
   belongs_to :portfolio_item
   has_many :progress_messages
   after_initialize :set_defaults, unless: :persisted?
 
-  NON_DATE_ATTRIBUTES = %w(order_id service_plan_ref portfolio_item_id state service_parameters provider_control_parameters external_ref)
+  NON_DATE_ATTRIBUTES = %w[order_id service_plan_ref portfolio_item_id state service_parameters provider_control_parameters external_ref workflow_ref].freeze
```

Comment:
These constants are going to be removed as they are only used by `as_json` which was refactored out of these methods in https://github.com/ManageIQ/catalog-api/pull/158

Also, I don't think we should be returning `workflow_ref` as part of the json payload either.  It seems like internal data.

Reply:
I was wondering if we were gonna rip out this custom `as_json` stuff. Should I remove it or will there be a future story to remove it?

Reply:
The PR I linked above removes the `as_json` from the models as well as the constants.